### PR TITLE
BZ#189485 Added no-support statements, edits for clarity

### PR DESCRIPTION
**Pull request by ctauchen**
This PR:
* fixes [BZ#1898485](https://bugzilla.redhat.com/show_bug.cgi?id=1898485)
* adds a few edits for clarity
* applies to version 4.6+
* [direct preview link](https://deploy-preview-32660--osdocs.netlify.app/openshift-enterprise/latest/nodes/edge/nodes-edge-remote-workers.html#nodes-edge-remote-workers-strategies_nodes-edge-remote-workers)

### CODE REVIEW

**Comment by ctauchen** [comment]
@openshift/team-documentation for peer review please. Please can someone also add 'Peer Review Needed', 'branch/enterprise-4.6', 'branch/enterprise-4.7', and 'branch/enterprise-4.8' labels and 'Next Release' milestone? Thanks!

File: modules/nodes-edge-remote-workers-strategies.adoc [hunk 1]
```diff
@@ -112,11 +112,11 @@ spec:
...
+* The `node-monitor-grace-period` parameter specifies how long {product-title} waits after a node associated with a `MachineConfig` object is marked `Unhealthy` if the controller manager does not receive the node heartbeat. Workloads on the node continue to run after this time. If the remote worker node rejoins the cluster after `node-monitor-grace-period` expires, pods continue to run. New pods can be scheduled to that node. The `node-monitor-grace-period` interval is `40s`. The `node-status-update-frequency` value must be lower than the `node-monitor-grace-period` value. Modifying the `node-monitor-grace-period` interval is not supported.
```
Comment (lbarbeevargas) [inline modules/nodes-edge-remote-workers-strategies.adoc:117]: What do you think about adding "the" back in front of `node-monitor-grace-period` for readability?

> If the remote worker node rejoins the cluster after _the_ `node-monitor-grace-period` expires, pods continue to run.

File: modules/nodes-edge-remote-workers-strategies.adoc [hunk 2]
```diff
@@ -112,11 +112,11 @@ spec:
...
 
-* The `pod-eviction-timeout` parameter specifies the amount of time {product-title} waits after marking a node that is associated with a `MachineConfig` object as `Unreachable` to start marking pods for eviction. Evicted pods are rescheduled on other nodes. If the remote worker node rejoins the cluster after `pod-eviction-timeout` expires, the pods running on the remote worker node are terminated because the node controller has evicted the pods on-premise. Pods can then be rescheduled to that node. The `pod-eviction-timeout` period is `5m0s`.
+* The `pod-eviction-timeout` parameter specifies the amount of time {product-title} waits after marking a node that is associated with a `MachineConfig` object as `Unreachable` to start marking pods for eviction. Evicted pods are rescheduled on other nodes. If the remote worker node rejoins the cluster after `pod-eviction-timeout` expires, the pods running on the remote worker node are terminated because the node controller has evicted the pods on-premise. Pods can then be rescheduled to that node. The `pod-eviction-timeout` interval is `5m0s`. Modifying the `pod-eviction-timeout` interval is not supported.
```
Comment (lbarbeevargas) [inline modules/nodes-edge-remote-workers-strategies.adoc:119]: Same question/suggestion to add "the" in front of `pod-eviction-timeout` for readability:

> If the remote worker node rejoins the cluster after _the_ `pod-eviction-timeout` expires ...

**Review by lbarbeevargas — COMMENTED** [review]
These updates LGTM! Just a few small suggestions for readability. What do you think about putting the "not supported" wording in a note/admonition after the two bullets? For example:

> [NOTE]
> Modifying the `pod-eviction-timeout` and `node-monitor-grace-period` intervals is not supported.

File: modules/nodes-edge-remote-workers-strategies.adoc (hunk 1 above)
Comment (ctauchen) [reply modules/nodes-edge-remote-workers-strategies.adoc]: Hi, @lbarbeevargas! Thanks for the comments!

I moved the no-support lines into the note as you suggested. That definitely looks better.

Not so sure about adding a definite article before an object name, but I can't find anywhere in our guides that discusses this specifically. It's an interesting case.

I think a good rule of thumb is to treat an object name as a proper noun, like Chris, Mississippi, or, say, rk86sh5.

Consider:

(+) Assign a value to the `chris` parameter.
(+) Assign a value to `chris`. 
(-) Assign a value to the `chris`.*
\* _'The' is clearly out of place here._

(+) Assign a value to the `rk86sh5` parameter.
(+) Assign a value to `rk86sh5`. 
(-) Assign a value to the `rk86sh5`.*
\* _'The' is clearly out of place here, too._

(+) Assign a value to the `node-monitor-grace-period` parameter.
(+?) Assign a value to `node-monitor-grace-period`.
(-?) Assign a value to the `node-monitor-grace-period`.
_The name of this parameter sounds like a normal phrase, so we're tuned in to hearing correct syntax. It sounds better to hear the definite article. (In English, at least: localized versions will treat this just like rk86sh5.)_

Does that explanation make sense?

Comment (lbarbeevargas) [reply modules/nodes-edge-remote-workers-strategies.adoc]: Thank you for the explanations on including or not including a definitive article to these examples. Sounds good to keep them as is! 👍

**Comment by sunilcio** [comment]
LGTM

**Pull request merged**